portal/src/texts/core/pattern.md

```diff
@@ -19,3 +19,29 @@ Måten vi bruker, fjerner eller legger til data i et datasett, kan vi også stan
 ### Overflytsregler
 
 Når en tekst er for lang for et bestemt område, kan vi bestemme hvordan vi vil at dette skal vises. Vi kan for eksempel forkorte informasjonen og vise en ellipse. I Jøkul finner du blant annet mønstre for hvordan vi forkorter søkebaner og lange URL-er.
+
+
+# Skjemadesign
```

Comment:
Kunne vi lagd en `tldr:` på dette?

Reply:
Hvordan lager vi en tldr:? Forenkler teksten? 
Ref eksempelet som ble vist i AHF-demo med søkefelt og en lang adresse som resultat: Ser ikke for meg at en ellipse vil være en fullgod løsning der. Må se på andre patterns i slike tilfeller.

Reply:
@Steinop, jeg tror @lfbergee refererer til avsnittet om skjemadesign når han vil ha en tl;dr 😊 Det høres for øvrig ut som en god idé: Ha en kort statement på siden om mønstre, med link videre til delen om skjemadesign (når vi får restrukturert portalen)